### PR TITLE
Community Blog post on SwC localisation efforts in Ukrainian language.

### PR DESCRIPTION
**Pull request by OscarSiba**
Hi @olexandr-konovalov, here is our republished version of your post.

### CODE REVIEW

**Review by kariljordan — APPROVED** [review]
@OscarSiba this post looks great! So glad we were able to cross-post it.

File: _posts/2024/05/2024-05-17-software-carpentries-translation-efforts-in-ukrain.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+layout: page
+authors: ["Dr. Olexandr Konovalov"]
+teaser: "An overview of the current state of the Ukrainian language localisation project initiated in 2022."
+title: "Translating Software Carpentry lessons into Ukrainian language: a status update"
+date: 2024-05-17
+time: "09:00:00"
+tags: ["Software Carpentry", "Internationalisation", "Equity and Inclusion", "Community"]
+---
+
+_This blog post was orginally published on Dr. Olexandr Konovalov's website as ["Software Carpentry for Ukraine: status update"](https://olexandr-konovalov.github.io/posts/2024/04/18/ukrainian-carpentry/). Minor modifciations in style have been made to the original so that this version is in keeping with The Carpentries blog post conventions._
```
Comment (olexandr-konovalov) [inline _posts/2024/05/2024-05-17-software-carpentries-translation-efforts-in-ukrain.md]: typo: "modifciations"

**Review by olexandr-konovalov — APPROVED** [review]
@OscarSiba - just one typo to fix, everything else looks fine. Thank you!

**Review by olexandr-konovalov — CHANGES_REQUESTED** [review]
@OscarSiba can we also rename the file, not to truncating "ukrain", please? Could end with "ukrainian" for more readable URL? If you want to shorten the URL to keep it within some length limit, maybe remove "efforts"?

File: _posts/2024/05/2024-05-17-software-carpentries-translation-efforts-in-ukrain.md (hunk 1 above)
Comment (OscarSiba) [reply _posts/2024/05/2024-05-17-software-carpentries-translation-efforts-in-ukrain.md]: Thank you @olexandr-konovalov, will update.

**Comment by olexandr-konovalov** [comment]
@OscarSiba perfect, thanks!

**Pull request merged**